mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -51,7 +51,7 @@ private[ml] trait CrossValidatorParams extends ValidatorParams {
   /** @group getParam */
   def getNumFolds: Int = $(numFolds)
 
-  setDefault(numFolds -> 3)
+  setDefault(numFolds -> 3, numParallelEval -> 1)
```

Comment:
May make more sense to put the `setDefault` call in the parent trait `ValidatorParams`